### ISSUE TITLE
Can't edit page while using WPMU and Domain Mapping

### ISSUE THREAD

**Issue by lippoliv** [pojome/elementor#473]
While using Elementor in an WordPress Multisite / Blog Network all is OK. But as soon as you use Domain Mapping and set an new Primary Domain, you can't use Elementor to edit your pages anymore.

Here's my Domain Mapping:

![unbenannt](https://cloud.githubusercontent.com/assets/2561859/18124405/698b67ca-6f72-11e6-9952-e13d4c9431d1.PNG)

This is the Link of the Button "Edit with Elementor":  http://lipperts-web.de/fi-addons/forcessl/?elementor and through the Domain Mapping I will be redirected to http://fi-addons.de/forcessl/?elementor. But Elementor doesn't start. Also I don't have an "Edit with Elementor"-Button in the Admin-Bar on this page.

Info: While switching Primary Domain back to the suggested WP Domain, Elementor works like a charme ;)

**Comment by joshmarom** [comment]
dup #423

**Comment by felipe-paniago-movile** [comment]
I'm using WPML to have a website with different languages; And when activated the subdomain, Elementor didn't work only for it;

example: mysite.com (Elementor works)
                us.mysite.com (Elementor don't open the editor)

Any tip?

**Comment by lippoliv** [comment]
Any updates? Would be nice to launch my page coming days

**Comment by lippoliv** [comment]
Hey there,
good news: Today I started Debugging (due to customer did not send documents to do my work):

`if ( ! version_compare( PHP_VERSION, '5.4', '>=' ) ) {` in `elementor.php` failed. I was confused. But now everything makes sense.
I host my Website via Plesk Panel. The Abonnement has configured PHP 5.3.29 (older Abonnement, a lot of different Apps, I have fear to Change Version for all, doesn't matter).
The Blog-Network Domain lipperts-web.de has configured PHP 5.4.16.

And now the human error is added to the whole issue: Elementor of Course is working on WP Blog Network Domain as of it is the accepted PHP Version. I added the new Domains fi-addons.de / fi-addons.com to the Abonnement, lead them into the right Directory, but doesn't set an PHP Version. So theese Domains using Abonnements PHP Version wich is 5.3.29.

And boom: Version compare Fails.

And here is why the Admin Notice "failes": As of the Front-End is on fi-addons.de with an bad PHP Version the Admin-Notice is tryed to added, but WP don't Show them, because we're still in front end. While moving back to Backend, we aren't any longer on fi-addons.de (bad PHP Version) but on lipperts-web.de (lipperts-web.de/fi-addons/wp-admin/index.php) wich has an accepted PHP Version and no Admin notice will be added (wich WordPress now would render as of we're in Backend).


I set the right PHP Version to the front-end Domain and boom: **Elementor is working with WPMU and Domain Mapping**.
